### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by N07070**
Changed the format of the date when saving the snaps. Updated it to fit the ISO 8601 standard ( https://en.wikipedia.org/wiki/ISO_8601 ) and RCF 3339 ( https://tools.ietf.org/html/rfc3339 )
Also, added the format of saving to date - user, as it is easier to read in a folder. ( It is also easier to grep all of the pics of a user than to get the time ).

### CODE REVIEW

File: snapchat_bots/utils.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def create_temporary_file(suffix):
 
 def save_snap(snap):
     now = datetime.datetime.now()
-    filename = '%s-%s.%s.%s-%s:%s:%s%s' % (snap.sender, now.month, now.day, now.year, now.hour, now.minute, now.second, snap.file.name[-4:])
+    filename = '%s-%s-%s %s:%s:%s x %s%s' % (now.year, now.month, now.day, now.hour, now.minute, now.second, snap.sender, snap.file.name[-4:])
```
Comment (N07070) [inline snapchat_bots/utils.py]: Changed the disposition of the filename, thus making it more practical to read them in a chronological order.

**Comment by agermanidis** [comment]
Hey, can you fix this is so it works in conjunction with your other pull request? Also, perhaps you should consider a format without spaces like 2015-02-25T16:42:11Z (still ISO 8601)?

Cheers,
-A

**Comment by N07070** [comment]
I think I will un-publish this pull request, and re-write the code by forking the last commit of utils.py.

**Comment by agermanidis** [comment]
:+1:

**Comment by N07070** [comment]
Done, you can refuse this one.

**Pull request closed without merging**